webhooktranslator/requirements.txt

```diff
@@ -0,0 +1,3 @@
+jira==1.0.10
```

Comment:
This file is not accurate or required. Deps have been inlined into setup.py. I'll remove this after the first round of reviews is complete.